TinCan/ActivityDefinition.cs

```diff
@@ -85,6 +146,70 @@ public ActivityDefinition(JObject jobj)
             {
                 result.Add("extensions", extensions.ToJObject(version));
             }
+            if (interactionType != null)
+            {
+                result.Add("interactionType", interactionType.ToString());
+            }
+            if (correctResponsesPattern != null && correctResponsesPattern.Count > 0)
+            {
+                var jcorrectResponsesPattern = new JArray();
```

Comment:
Would prefer to avoid the use of `var`. I know this is subjective, AFAIK we aren't using it anywhere else in the code base.